components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/Oauth2ScopeUtils.java

```diff
@@ -331,7 +333,7 @@ public static boolean isSystemLevelInternalSystemScopeManagementEnabled() {
      * @return String array with internal scopes. Return an empty array if there's not any internal scopes in the
      * given scopes array.
      */
-    public static String[] getRequestedScopes(String[] scopes) {
+    public static String[] getRequestedInternalScopes(String[] scopes) {
```

Comment:
Changing a public method might break dependent code.

Deprecate and introduce a new method if needed